### DISCUSSION TITLE
Yellow surrounding over trace

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#998] [General]
Hello
What is this 1cm wide yellow surrounding around the circuit for and how to remove it?
Clicking on Affix the entire circuit icon removes it but it comes back quickly
![YellowSurrounding](https://user-images.githubusercontent.com/4082252/219853205-16cf593b-bf7d-4690-ae09-1b9918b0a2ff.jpg)

**Comment by FJBDev** [comment]
> Hello What is this 1cm wide yellow surrounding around the circuit for and how to remove it?

It's the _"chart slider path"_ that you can remove in the [2DMap slider](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/show-tours/tour-map/2d-track-options)

**Reply by gringojack** [reply]
Thank you for this info
